lib/senders/index.js

```diff
@@ -96,16 +96,35 @@ module.exports = function (log, config, error, db, translator, sender) {
 
     var getSafeMailer = BOUNCES_ENABLED ? bounceGatedMailer : noopMailer
 
+    function getVerifiedSecondaryEmails(emailData) {
+      return emailData.reduce(function(list, email) {
+        if (! email.isPrimary && email.isVerified) {
+          list.push(email.email)
+        }
+        return list
+      }, [])
+    }
+
+    function getSecondaryEmails(emailData) {
+      return emailData.reduce(function(list, email) {
+        if (! email.isPrimary) {
+          list.push(email.email)
+        }
+        return list
+      }, [])
+    }
+
     senders.email = {
-      sendVerifyCode: function (account, code, opts) {
-        return getSafeMailer(account.email)
+      sendVerifyCode: function (emailData, account, opts) {
```

Comment:
I'm deep into bike-shedding territory here but I'm not a *huge* fan of the name `emailData` for these arguments, it doesn't quite suggest this-is-an-array-of-email-addresses to me. If it was *my* bike shed, I'd paint it more like `emails` or `emailAddresses` or something. :smile: